modules/social_features/social_user_export/src/ExportUser.php

```diff
@@ -34,6 +34,9 @@ public static function exportUserOperation(UserInterface $entity, array &$contex
       $headers = [
         t('ID'),
         t('UUID'),
+        t('FirstName'),
```

Comment:
Lets use the same proper formatting of the other fields and make this `First name`